### ISSUE TITLE
No style for blockquotes

### ISSUE THREAD

**Issue by anwalkers** [digitalcraftsman/hugo-material-docs#22]
There is now style for blockquote. It is zeroed and then not styled. Maybe something like...

```
blockquote {
    padding: 0 20px;
    margin: 0 0 20px;
    font-size: inherit;
    border-left: 5px solid #eee;
}
```

**Comment by schlawiner** [comment]
I've noticed that as well.
A workaround is to define it like your example in a custom css as set in your config.toml.

**Comment by digitalcraftsman** [comment]
Fixed in 75ecc75cc16425c72adda67ab7f24af49ddb2644 with pull request originally made by @anwalkers.
